credentials_manager/lib/credentials_manager/account_manager.rb

```diff
@@ -117,7 +117,7 @@ def ask_for_login
         prompt_text += " (#{@note})" if @note
         prompt_text += ": "
         @user = ask(prompt_text) while @user.to_s.length == 0
-        # we return here, as only the username was asked for now, we'll get called for the pw again anyway
+        # we return here, as only the username was asked for now. We'll get called for the pw again anyway.
```

Comment:
This sentence is still a bit confusing 😇 So I would change it to something like...

```
# Returning here since only the username was asked for. This method will be called again when a password is needed.
```